sql-schema/282.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE `users` ADD `enabled` TINYINT NOT NULL DEFAULT '1';
```

Comment:
Hi,
Since we moved to Laravel, we cannot use the <id>.sql files. You cannot change them at all. You need to use Laravel migrations.
https://laravel.com/docs/5.8/migrations

You can have a look at this folder for existing migration classes: 
./database/migrations

Reply:
@PipoCanaja 
migration changed :-)

Reply:
Looks better now :) Travis will still complain because the db schema is not up to date. 
Can be done by running : 
./script/build-schema.php
That should help Travis to be happy.

Reply:
i tried it, but it doesn't work at me:

```
PHP Fatal error:  Uncaught Error: Call to undefined function LibreNMS\Util\app() in /opt/librenms/LibreNMS/Util/Laravel.php:71
Stack trace:
#0 /opt/librenms/LibreNMS/Util/Laravel.php(38): LibreNMS\Util\Laravel::isBooted()
#1 /opt/librenms/includes/init.php(96): LibreNMS\Util\Laravel::bootCli()
#2 /opt/librenms/includes/sql-schema/update.php(36): require('/opt/librenms/i...')
#3 /opt/librenms/scripts/build-schema.php(7): require('/opt/librenms/i...')
#4 {main}
  thrown in /opt/librenms/LibreNMS/Util/Laravel.php on line 71

Fatal error: Uncaught Error: Call to undefined function LibreNMS\Util\app() in /opt/librenms/LibreNMS/Util/Laravel.php:71
Stack trace:
#0 /opt/librenms/LibreNMS/Util/Laravel.php(38): LibreNMS\Util\Laravel::isBooted()
#1 /opt/librenms/includes/init.php(96): LibreNMS\Util\Laravel::bootCli()
#2 /opt/librenms/includes/sql-schema/update.php(36): require('/opt/librenms/i...')
#3 /opt/librenms/scripts/build-schema.php(7): require('/opt/librenms/i...')
#4 {main}
  thrown in /opt/librenms/LibreNMS/Util/Laravel.php on line 71
```

Reply:
what does ./validate.php says ? does the daily script run regularly ? This script should run smoothly and update the misc/db_schema.yaml file

Reply:
@PipoCanaja 
i run this now in my running development/testing instance of librenms, and it worked.
Script missed some laravel stuff which will be installed on configuration.

i hope now everything is fine.
by the way. I've got 2 Application enhancements since weeks as pull request waiting for merge. Is there a chance to speed this a little bit up? ;-)
Certificate validation check #10354 
Seafile Monitoring #10465

Reply:
For current PR, you still need to remove the changes you did on 282.sql. These files cannot be changed anymore.

For the other PRs, we lack admins to review them, we lack testers to test them, we don't always have the right environnement to test them  etc etc. We do our best on our free time, which is usually not enough.

Reply:
@PipoCanaja 
282.sql removed